src/test/java/org/folio/rest/impl/TenantSampleDataTest.java

```diff
@@ -119,11 +123,32 @@ public void testPartialSampleDataLoading() throws MalformedURLException {
     }
   }
 
+  @Test
+  public void testLoadReferenceData() throws MalformedURLException {
+    logger.info("load only Reference Data");
+    try {
+      JsonObject jsonBody = TenantApiTestUtil.prepareTenantBody(false, true, false);
```

Comment:
you can override the prepareTenant method, so that the repetition of these 2 statements in the file can be avoided